pkg/utils/version.go

```diff
@@ -1,6 +1,7 @@
 package utils
 
 var (
-	Version = "dev"
-	Sha     = "HEAD"
+	Version  = "dev"
+	Sha      = "HEAD"
+	Datetime = "dev"
```

Comment:
I was contemplating putting a default build date that was `time.Now()`, but thought it was better for the dev environment to default to `dev` like the version.